5.5/docker-entrypoint.sh

```diff
@@ -1,13 +1,25 @@
 #!/bin/bash
 set -e
 
+get_option () {
+    local section=$1
+    local option=$2
+    local default=$3
+    ret=$(/usr/local/mysql/bin/my_print_defaults $section | grep '^--'${option}'=' | cut -d= -f2-)
```

Comment:
Why not just rely on the `PATH` for finding the right `my_print_defaults` binary?